xml/smb_native_os.xml

```diff
@@ -358,6 +358,32 @@
       <param pos="1" name="os.build"/>
    </fingerprint>
 
+   <!-- Windows 10 Preview -->
+   <fingerprint pattern="^Windows 10 (\w+|\w+ \w+|\w+ \w+ \w+) Insider Preview (\d+)$">
+      <description>Windows 10 Enterprise Insider Preview</description>
+      <example os.build="10130" os.edition="Enterprise">Windows 10 Enterprise Insider Preview 10130</example>
+      <param pos="0" name="os.certainty" value="1.0"/>
+      <param pos="0" name="os.vendor" value="Microsoft"/>
+      <param pos="0" name="os.product" value="Windows 10"/>
+      <param pos="1" name="os.edition"/>
+      <param pos="2" name="os.build"/>
+   </fingerprint>
+
+   <fingerprint pattern="^Windows 10 (\w+|\w+ \w+|\w+ \w+ \w+) (\d+)$">
```

Comment:
Minor, but I think this would have been better off combined with the previous fingerprint -- they both assert a Windows 10 fingerprint, the only difference is the first one includes support for `Insider Preview` in the regex.  Simply make that part optional.